skyportal/handlers/api/photometry.py

```diff
@@ -401,6 +410,14 @@ def post(self):
 
         groupquery = GroupPhotometry.__table__.insert()
         params = []
+        if group_ids == "all":
+            group_ids = [
+                s.group_id
+                for s in DBSession()
+                .query(Source)
+                .filter(Source.obj_id == packet["obj_id"])
+                .all()
+            ]
```

Comment:
We want this to truly be all groups (maybe sitewide group), not just groups an object is saved to. Maybe we should just use the sitewide_group here?

Reply:
Okay, but in the issue you wrote "shared with any group with access to the source", which sounds like all groups it's saved to

Reply:
Done